### PR TITLE
Init 2019 Roadmap

### PR DESCRIPTION
**Pull request by cheyang**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/arena/142)

### CODE REVIEW

File: ROADMAP_cn.md [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
 
-#### 稳定性/可靠性
+* 适配不同版本:
+	* v1aphla2 和 v1 TFJob
+	* v1alpha1 和 v1aphla2 MPIJob
 
-- 端到端测试
-- 单元测试
-- 自动生成 arena docker镜像
+目标: 重构代码并且提升自动化测试能力进而能实现告诉迭代
```
Comment (denverdino) [inline ROADMAP_cn.md]: 高速

File: ROADMAP.md [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
-#### Features
+Objectives: "Make Arena easily to be integrated with External System."
 
-- Enhance Training Job
-  - Move to MPI-operator
-  - Set default CPU/Memory limit according to different types of training: tf-operator, MPI-operator
-  - Support Gang Scheuler
-  - Pytorch-operator
+* Integration
+	* Provide the Java, Python and C++ API for system interaction
```
Comment (wsxiaozhang) [inline ROADMAP.md]: we'd better release SDK for integration first, instead of API

File: ROADMAP.md [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
-- Training History Management
-  - Use CRD to manage the training history
+Objectives: "Simplify the user experience of the data scientists and provide a low barrier to handle different kind of  training jobs and serve different models."
 
-- Integrate with data
-  
-- Muti-tenancy
+* High Level Interfaces:
+	* Submit and manage Flink Jobs
+	* Submit and manage Spark Jobs
```
Comment (wsxiaozhang) [inline ROADMAP.md]: Submit and manage other data processing and machine learning jobs, like Spark, Flink, XDL

Comment (cheyang) [reply ROADMAP.md]: Updated.

File: ROADMAP_cn.md (hunk 1 above)
Comment (cheyang) [reply ROADMAP_cn.md]: Updated.

**Pull request merged**